packages/ui/src/GoogleButton.tsx

```diff
@@ -37,7 +39,7 @@ export function GoogleButton(props: GoogleButtonProps): JSX.Element | null {
       google.accounts.id.initialize({
         client_id: googleClientId,
         callback: (response: GoogleCredentialResponse) =>
-          medplum.startGoogleLogin(response).then(props.handleAuthResponse),
+          medplum.startGoogleLogin({ ...response, projectId }).then(handleAuthResponse),
```

Comment:
This change might not be necessary.... 

The GoogleCredentialResponse includes a clientId.  That clientId is either (1) the main Medplum client or (2) a custom domain client.  The proposed model is that custom domain clients are always associated with a single Medplum project (?).